{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/monitoring/SetUpDataMonitoringForInferenceTable.py

```diff
@@ -0,0 +1,135 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC
+# MAGIC # Data Monitoring for Inference Table
+# MAGIC see monitoring/README.md for details
+
+# COMMAND ----------
+
+# Install the Data Monitoring client library
+# TODO: Fill in wheel URL
+%pip install "{wheel_URL}"
```

Comment:
Do we want to make this a DB widget instead to make it more obvious? Or do we not expect this notebook to be opened in Databricks?

Reply:
sure